mail_tracking_mailgun/models/res_partner.py

```diff
@@ -26,10 +26,13 @@ def _email_bounced_set(self, reason, event):
         for partner in self:
             if not partner.email:
                 continue
-            body = _('Email has been bounced: %s\n'
-                     'Reason: %s\n'
-                     'Event: %s') % (partner.email, reason,
-                                     event and event.id or _('unknown'))
+            body = _(
+                'Email has been bounced: %s\n'
+                'Reason: %s\n'
+                'Event: '
+                '<a href=# data-oe-model=mail.tracking.event data-oe-id=%d>'
+                '%s</a>') % (partner.email, reason, event and event.id or 0,
```

Comment:
Apart from what @pedrobaeza said, also here you can use:

```suggestion
                '%s</a>') % (partner.email, reason, event.id or 0,
```